src/lib/normalize-url.js

```diff
@@ -0,0 +1,40 @@
+
+// Mobile apps give us full URL with domain and protocol.
+// Web apps could also initilize with a full URL
+// but they usually don't, and we the use the 
+// domain given by the stack through a DOM node
+// in the HTML page (depite it not being a full URL).
+//
+// This function normalize these different cases
+// and give a full URL object in response.
+// 
+// The `secure` parameter, if given, always has the
+// priority over the one from the URL.
+// We use a secure protocol by default if we can't decide.
+
+export default function normalizeURL(cozyURL, ssl) {
```

Comment:
```suggestion
export default function normalizeURL(cozyURL, secure = true) {
```

makes it clearer that we default to true for the secure parameter.

I would rename ssl to secure since https can be over ssl or tls.